src/prefect/utilities/annotations.py

```diff
@@ -41,7 +41,7 @@ class unmapped(BaseAnnotation[T]):
 
     def __getitem__(self, _) -> T:
         # Internally, this acts as an infinite array where all items are the same value
-        return self.unwrap()
+        return super().__getitem__(_)
```

Comment:
mm this isn't what we want to do  here `__getitem__` should return the value no matter the index. Is this being called by `self.value`?

Reply:
Right, I think you might have to refresh to see the new changes.